engine/src/rgbscriptscache.cpp

```diff
@@ -69,6 +69,11 @@ bool RGBScriptsCache::load(const QDir& dir)
 
     foreach (QString file, dir.entryList())
     {
+        if (!file.toLower().endsWith(".js") || file.endsWith("devtool.js"))
```

Comment:
As you can see here
https://github.com/mcallegari/qlcplus/blob/master/resources/rgbscripts/rgbscripts.pro
devtool.js is not installed, so I don't see a reason to check for it in the code.

Reply:
Hello Massimo, for the application image, this is true. Nevertheless, people customizing their scripts in the user directory, will have a copy of the devtool.js around in the user script directory.
I will nevertheless remove the check as you requested. If you can follow my thoughts, I can add it again as it only runs once when the cache loads the scripts and thus should not pose a performance issue.